accounts.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'transactions.rb'
+
+class Accounts
+  attr_accessor :name, :currency, :balance, :nature, :transactions
+  def initialize(name, currency, balance, nature)
+    @name         = name
+    @currency     = currency
+    @balance      = balance
+    @nature       = nature
+    @transactions = [transactions]
```

Comment:
Ummm? where do you get the `transactions` variable from?

Reply:
leave @transactions empty array `[]`